runtimevar/constantvar/constantvar_test.go

```diff
@@ -34,7 +34,7 @@ func newHarness(t *testing.T) (drivertest.Harness, error) {
 	return &harness{vars: map[string][]byte{}}, nil
 }
 
-func (h *harness) MakeWatcher(ctx context.Context, name string, decoder *runtimevar.Decoder, wait time.Duration) (driver.Watcher, error) {
```

Comment:
Just curious, was wait not used at all before?

Reply:
Yes, it was used before; the conformance tests passed different values (`shortWait` or `defaultWait`) depending on the test. Now that they use the driver interface explicitly, that's no longer needed.